sound/soc/sof/intel/skl.c

```diff
@@ -52,6 +52,10 @@ const struct snd_sof_dsp_ops sof_skl_ops = {
 	.send_msg	= hda_dsp_ipc_send_msg,
 	.fw_ready	= hda_dsp_ipc_fw_ready,
 
+	.ipc_read	= hda_ipc_read,
+	.ipc_read_pcm_stream = hda_ipc_read_pcm_stream,
+	.ipc_read_pcm_params = hda_ipc_read_pcm_params,
+
```

Comment:
@lyakh the commit title seems odd. what are we removing here?

Reply:
this is the skl part of the "removing." For skl it really means the second part of the commit message: "move them from the SOF core to Intel-specific drivers." TBH, this is a result of a single patch split in "what goes into the mainline" and "pieces for drivers, that aren't going to the mainline for now." I reused commit messages over such splits, all the more, that those commits will be squashed anyway, as far as I understand. But if desired, I can engage my creativity some more...